usaspending_api/api_contracts/contracts/v2/references/award_types.md

```diff
@@ -0,0 +1,87 @@
+FORMAT 1A
+HOST: https://api.usaspending.gov
+
+# Award Types [/api/v2/references/award_types/]
+
+This endpoint returns a JSON object representing the grouping of award types.
+
+## GET    
++ Response 200 (application/json)
+    + Attributes (object)
+        + `results` (required, AwardTypes, fixed-type)
+    + Body
+            {
+                "results": {
```

Comment:
Is it beneficial for clients to have the data nested under `results`? It seems extraneous

Reply:
Pushed changes; thanks @tony-sappe